desktop-app/src/common/deviceList.ts

```diff
@@ -870,6 +870,47 @@ export const defaultDevices: Device[] = [
     isTouchCapable: false,
     isMobileCapable: false,
   },
+  {
+    id: '90004',
```

Comment:
Can you please fix the IDs to stick to the guidelines here: https://github.com/responsively-org/responsively-app/pull/1293/files#diff-1f757e572fe35320119d79efc7411be133df8ec132a1737e78a872eaa1720da8R16-R22